<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>themkhoanthu_form</class>
 <widget class="QMainWindow" name="themkhoanthu_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thêm khoản thu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(224, 243, 253);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);
font: 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Thêm thu nhập</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Số tiền:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ghi chú:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ghichu">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 234, 255);
font: 12pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chọn 1 mục tiêu tài chính:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_sotien">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>42</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>42</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;đ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ngày:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_return.png</normaloff>images/ic_return.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_muctieutc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTreeWidget" name="trw_khoanthu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>141</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Khoản thu</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Thu nhập chính</string>
     </property>
     <item>
      <property name="text">
       <string>Lương</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tiền thưởng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phụ cấp</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Từ hoạt động kinh doanh &amp; đầu tư</string>
     </property>
     <item>
      <property name="text">
       <string>Bán hàng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Đầu tư chứng khoán / cổ phiếu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lãi suất ngân hàng, tiền gửi tiết kiệm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Đầu tư bất động sản</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Tiền trợ cấp</string>
     </property>
     <item>
      <property name="text">
       <string>Trợ cấp từ gia đình</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quà tặng, tiền mừng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bảo hiểm xã hội, trợ cấp thất nghiệp, học bổng,...</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Thu nhập khác</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 234, 255);
font: 12pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Khoản thu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_luu">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Lưu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_capnhat">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Cập nhật</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_xoa">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Xóa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_huy">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 194, 209);</string>
       </property>
       <property name="text">
        <string>Hủy bỏ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tbl_table">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>txt_ghichu</zorder>
   <zorder>label_6</zorder>
   <zorder>dateEdit</zorder>
   <zorder>label_7</zorder>
   <zorder>label_3</zorder>
   <zorder>btn_return</zorder>
   <zorder>cbb_muctieutc</zorder>
   <zorder>trw_khoanthu</zorder>
   <zorder>label_4</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>tbl_table</zorder>
   <zorder>txt_sotien</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
